doc/controls/CardAndCardContentControl.md

```diff
@@ -0,0 +1,102 @@
+# Card & CardContentControl
+
+## Summary
```

Comment:
Can we add screenshots?

Reply:
Yes @jeromelaban but we will add them as the same time that the v2 styles will to be done for Card next week. (cc @kazo0)